utils/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required (VERSION 3.18)
 project (HDF5_UTILS C)
 
-if (BUILD_TESTING)
+if (NOT HDF5_EXTERNALLY_CONFIGURED AND BUILD_TESTING)
   add_subdirectory (test)
 endif ()
```

Comment:
I think one could also think of only building the tests if the utils are built in the first place. Building only the tests doesn't seem to make much sense :thinking: :eyes: